### PR TITLE
rework of concepts course materials

### PR DESCRIPTION
**Pull request by lakikowolfe**

### CODE REVIEW

**Comment by k8hertweck** [comment]
You've got `images/.DS_Store` lurking in there, which isn't horrible, but would be good to remove. Add `**.DS_Store` to `.gitignore` and remove this file from the history as well; more info [here](https://stackoverflow.com/questions/18393498/gitignore-all-the-ds-store-files-in-every-folder-and-subfolder). I'm also filing an issue in the template repo to include a standard .gitignore file in our repos so we can avoid this in the future

File: README.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ This four week course is designed to introduce attendees to central concepts in
 
 ## Schedule
 
-- Week 1: Introduction and Conceptual Overview; Machine Learning and Experimental Design
-- Week 2: Supervised Learning via Regression
-- Week 3: Supervised Learning via Classification
-- Week 4: Unsupervised Learning via Dimensionality Reduction, Clustering, and Transfer Learning
+- Class 1: Introduction and Conceptual Overview of Machine Learning Concepts
```
Comment (k8hertweck) [inline README.md]: I like adding links to the markdown file in each line, so that clicking on Class 1 takes you to the file.

File: class1.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
+
+## Concepts in Machine Learning Class 1: Introduction to Machine Learning
+
+### Objectives
+
+Welcome to Concepts in Machine Learning from fredhutch.io! This course is designed for researchers who are interested in machine learning, and assumes no prior programming or statistics experience. There are no prerequisites for this course.
+
+This course will introduce you to machine learning and give a broad overview of supervised and unsupervised machine learning methods. The goal of this course is to provide some conceptual background for our Machine Learning in R and Python series. Further, researchers who complete this course should have an understanding of how machine learning can be applied in biology research.
```
Comment (k8hertweck) [inline class1.md]: introduce you to machine learning and focus specifically on supervised and unsupervised machine learning methods.

Comment (k8hertweck) [inline class1.md]: I've been trying to use consistent labelling for courses, so refer to them as "Intermediate R and Python courses in Machine Learning"

Switch the order of the second and third sentences here.

File: class1.md [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+
+By the end of this course you should be able to:   
+
+* Identify how machine learning can be applied in biology research and form a problem statement.
```
Comment (k8hertweck) [inline class1.md]: On reading this again with the rest of the course objectives, I'd remove "and form a problem statement" because it seems thrown in there and doesn't have much context...you could also add an additional explanatory clause (form problem statements to...) but then I think it would be too long?

File: class1.md [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+* Apply the concepts of exploratory data analysis, experimental design, and ethics machine learning.
```
Comment (k8hertweck) [inline class1.md]: ethics in machine learning?

File: class1.md [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+* Magic
+* A secret sauce
+* Seasoning to be sprinkled liberally over a mundane application
+* A substitute for a good understanding of your problem
+
+### What **is** machine learning?
+
+Recent advances in compute power, cloud technologies, data collection and data generation have thrust machine learning into the mainstream. Increasingly machine learning is looked to as a tool to solve Big Complex Problems like cancer. Often though the term machine learning is used as a sort of black box, but what actually is machine learning?
+
+Let’s orient ourselves to the terms I mentioned in the previous slides that you may be familiar with (ML, AI, Deep learning) to get a sense of how things fit together.
```
Comment (k8hertweck) [inline class1.md]: slides?

Comment (k8hertweck) [reply class1.md]: also you need to define these acronyms. Either add them as parenthetical when you first describe them above, or define them here.

File: class1.md [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+
+<p align="center">
+  <img align="middle" width="500" src="images/aiMlDl.png">
+</p>
+
+* AI is a term that embodies anything that enables computers to act more like humans.
```
Comment (k8hertweck) [inline class1.md]: It's confusing that this is a nested list, since machine learning is actually a part of AI, etc. Maybe keep the three main points (AI, ML, deep learning) and turn the bullet points below them into sentences. For example:

* AI is a term that embodies anything that enables computers to act more like humans.
* Machine learning is a field of study within AI, similar to how molecular biology is a specific focus in biology. The focus of this field is on the extraction of patterns from large datasets.
* Deep learning is an example of a collection of similar machine learning methods.
...

File: class1.md [hunk 7]
```diff
@@ -0,0 +1,261 @@
...
+
+* Machine learning is a kind of algorithm.
+  * Algorithm - A finite sequence of well-defined, computer-implementable instructions, typically to solve a class of problems or to perform a computation 
+   * Essentially it is a protocol for a computer
+* Machine learning incorporates data into a statistical model.
+* Machine learning improves with experience without human intervention.
+
+### Machine learning as code
+
+When we think of machine learning and how it applies to code we can consider machine learning to be a specific way of programming computers.
```
Comment (k8hertweck) [inline class1.md]: This sentence is confusing. I think what you mean is that machine learning principles are applied to data using computer code, and the computational methods involve specific characteristics?

File: class1.md [hunk 8]
```diff
@@ -0,0 +1,261 @@
...
+
+* Most programming isn’t built to learn - it’s procedural
+    * This is where a programmer builds out a set of rules. This set of rules is called an algorithm. This ruleset remains static unless the coder intervenes and specifies more rules. 
+* In machine learning the algorithm is also designed and coded by a human. However, this kind of algorithm is able to learn from the data about specific parameters that will shape a statistical model for making predictions and inferences. The coder doesn’t necessarily know or control these parameters - the machine does.
+
+> **Machine learning is an algorithm (or set of rules) that self adjusts based on the large datasets that it is given**
+
+#### Challenge!
```
Comment (k8hertweck) [inline class1.md]: Include some context here: how does this challenge relate to the rest of the material?

I'd also format the header as Challenge-pbj so you can get a unique link to it (to share with learners) when you have more than one exercise present

File: class1.md [hunk 9]
```diff
@@ -0,0 +1,261 @@
...
+
+1. Write an algorithm/protocol/ruleset for making a PBJ sandwich
+2. Write an algorithm/protocol/ruleset for determining if an email is spam
+
+[click here to go to solutions page](https://github.com/fredhutchio/concepts_machine_learning/blob/master/solutions/class1_solutions.md#write-an-algorithmprotocolruleset-for-making-a-pbj-sandwich)
+
+### When to use machine learning
+
+Just like any other statistical method there are specific situations where applying machine learning is appropriate. Just as important as the algorithm itself is a solid understanding of experimental design, domain expertise in the problem area youre trying to solve, and an understanding of data governance and ethics.
```
Comment (k8hertweck) [inline class1.md]: Move the second sentence here to after the two paragraphs below.

File: class1.md [hunk 10]
```diff
@@ -0,0 +1,261 @@
...
+
+**You cannot code the rules:**
+* Tasks that cannot be solved using a simple, rule-based solution. 
+* When rules depend on too many factors and many of these rules overlap or need to be tuned very finely, it soon becomes difficult for a human to accurately code the rules. You can use ML to effectively solve this problem.
+* This is true for drawing conclusions about highly dimensional data.
+
+**You cannot scale:**
+* You might be able to manually recognize a few hundred emails and decide whether they are spam or not. However, this task becomes tedious for millions of emails. ML solutions are effective at handling large-scale problems.
+
+### Biology is suited for machine learning
```
Comment (k8hertweck) [inline class1.md]: This subheading seems to orient biology and ML awkwardly. I think we want to emphasize that ML is an appropriate approach for many biological questions. Otherwise, it makes it seem like you're trying to plan a wedding for bio and ML?

I'd also take it a step further and say biomedical data in particular can benefit from ML

File: class1.md [hunk 11]
```diff
@@ -0,0 +1,261 @@
...
+    * Our human brains cannot make sense of these huge integrated datasets or even just one of them.
+    * Example: a human looking at a series of bases would take years to make sense of the human genome. A computer can find patterns in our genome with relative ease.
+
+### How is machine learning applied?
+
+<img align="right" width="150" src="https://media.giphy.com/media/3oEduS0HyWuqR7kXL2/giphy.gif">
+
+Machine learning applications are all around us! Every time Spotify serves up a customized playlist or radio station based on the music that you like, Facebook loads your newsfeed, your phone autocorrects you, or you interact with a voice assitant like Siri, you are interacting with machine learning algorithms.
+
+Every-day applications:
```
Comment (k8hertweck) [inline class1.md]: I'd include a link here to a source (can be popular media) that talks about ML applied to one or more of these things

File: class1.md [hunk 12]
```diff
@@ -0,0 +1,261 @@
...
+* Search engines
+* Recommendation systems (Yelp, Netflix, Spotify)
+* Voice assistants
+* Social media feeds
+* Text prediction
+
+#### Machine learning techniques are becoming increasingly important to biological research.
```
Comment (k8hertweck) [inline class1.md]: My gut says bouncing back and forth between ML in general and as applied to biology is confusing. It makes more sense to me to order it:
- when to use ML in general (can't code rules, can't scale)
- common applications of ML (Siri, Spotify)
- why biological data can benefit from ML (can't code rules, can't scale)
- applications in biology

File: class1.md [hunk 13]
```diff
@@ -0,0 +1,261 @@
...
+
+**Inputs:** Features that we have collected and will use to make our prediction. Sometimes we will have datasets that have features that will not be included in downstream analysis. For example, in this dataset our inputs would be all of the columns except `patient_Id` and `cvd`. We would likely remove `pateint_Id` because a random identifier won't have any predictive power and `cvd` would not be included as an input since it is the target we are trying to predict.
+
+**Output:** The variable that we are trying to predict. In this case `cvd`.
+
+> #### Note!
+> The dataset above is from the package `cvdRiskData`. you can find more information [here](https://github.com/laderast/cvdRiskData). We will be using this dataset > as an example throughout this course.
```
Comment (k8hertweck) [inline class1.md]: Check formatting of these few lines (errant > and capitalization)

File: class1.md [hunk 14]
```diff
@@ -0,0 +1,261 @@
...
+* Prediction of protein secondary structure based on RNA-sequence.
+* Classification of histopathological images of breast tissue to identify malignant tumors.
+* Using gene expression data to classify patients into different clinical groups.
+
+Machine learning work that is happening here at Fred Hutch:
+* Gottardo lab - [automatic gating for flow cytometry data.](https://www.biorxiv.org/content/10.1101/702118v2)
+* Emily Silgard's data science group - [natural language processing to extract information from unstructured electronic medical records notes.](https://www.sciencedirect.com/science/article/abs/pii/S1532046419302217)
+* The Data Visualization Center - [single-cell visualization and clustering of RNA-sequencing data to find rare subtypes of cells that emerge in the mouse embryo.](https://oncoscape.v3.sttrcancer.org/atlas.gs.washington.edu.mouse.rna/landing)
+
+### How do these machine learning applications work?
```
Comment (k8hertweck) [inline class1.md]: This seems like an odd place to put this subheading. It seems to fit better after the anatomy section below (which doesn't mention supervised vs unsupervised)

File: class1.md [hunk 15]
```diff
@@ -0,0 +1,261 @@
...
+
+There are different types of machine learning and each have their own specific applications and requirements.
+
+Four major paradigms in machine learning:
```
Comment (k8hertweck) [inline class1.md]: I'd include brief definitions for each of these here, especially since you don't mention the last two at all anywhere else

File: class1.md [hunk 16]
```diff
@@ -0,0 +1,261 @@
...
+You might have noticed that this fairly simple dataset will not capture *all* of the complexities that lead to a diagnosis of cardiovascular disease, but it might capture enough variability to build a model and make accurate predictions. Only experimentation will show if this dataset is capable of creating an accurate and generalizable model.
+
+You'll notice that this dataset consists of variables that are very easy for someone to measure.
+
+Variables that are difficult to measure that might be related to cardiovascular disease:
+* Stress
+* Dietary habits
+* Sleep habits
+
+It's important to think critically about what features we want to include in our datasets, what features might be missing, and identify features that may introduce biases into the analysis. We will discuss exploratory data analysis and ethics as key pieces of the machine learning process in class 4 of this series.
```
Comment (k8hertweck) [inline class1.md]: series -> course

File: class1.md [hunk 17]
```diff
@@ -0,0 +1,261 @@
...
+* Semi-supervised
+* Reinforcement learning
+
+The focus of this course will be on supervised and unsupervised learning only, but it’s good to be aware that these are not the only types of machine learning. While increasingly complex machine learning algorithms are highlighted on TV and in the news, it's important to remember that the most commonly implemented form of machine learning is the simplest - supervised machine learning.
+
+### Anatomy of a machine learning problem
+
+The purpose of using machine learning is generally to use data make an inference or prediction. To do this, these algorithms access large datasets with many features. Datasets rarely, if ever, will capture all of the complex underlying variables that result in the Truth that we see in the world. Datasets merely capture the features that we think are related to the system that we are trying to uncover and that we can adequately measure and track.
+
+Here's an example training dataset for predicting whether or not a patient might be diagnosed with cardiovascular disease. This dataset shows paired input and output data together.
```
Comment (k8hertweck) [inline class1.md]: example training dataset -> peek at a portion of a dataset

current language implies that we'd only need two cases to make a prediction

File: class1.md [hunk 18]
```diff
@@ -0,0 +1,261 @@
...
+* **Supervised**
+* **Unsupervised**
+* Semi-supervised
+* Reinforcement learning
+
+The focus of this course will be on supervised and unsupervised learning only, but it’s good to be aware that these are not the only types of machine learning. While increasingly complex machine learning algorithms are highlighted on TV and in the news, it's important to remember that the most commonly implemented form of machine learning is the simplest - supervised machine learning.
+
+### Anatomy of a machine learning problem
+
+The purpose of using machine learning is generally to use data make an inference or prediction. To do this, these algorithms access large datasets with many features. Datasets rarely, if ever, will capture all of the complex underlying variables that result in the Truth that we see in the world. Datasets merely capture the features that we think are related to the system that we are trying to uncover and that we can adequately measure and track.
```
Comment (k8hertweck) [inline class1.md]: I know what you mean by Truth, but I would rephrase (and not capitalize) since it is hard to understand without the context of knowing you (and Thomas) :)

File: class1.md (hunk 13 above)
Comment (k8hertweck) [reply class1.md]: I'd also move this line up, so if people have questions they can access the data dictionary

File: class1.md [hunk 19]
```diff
@@ -0,0 +1,261 @@
...
+
+Here's an example training dataset for predicting whether or not a patient might be diagnosed with cardiovascular disease. This dataset shows paired input and output data together.
+
+| patient_Id    | age   | htn | treat | smoking | race     | t2d | gender | numAge | bmi | tchol| sbp | cvd |
+| ------------- | ----- | --- | ----- | ------- | -------- | --- | ------ | ------ | --- | ---- | --- | --- |
+| HHUID00076230 | 20-40 | Y   | Y     | N       | Asian/PI | N   | M      | 29     | 23  | 189  | 170 | N   |
+| HHUID00547835 | 70-90 | N   | Y     | N       | White    | Y   | M      | 72     | 35  | 178  | 118 | N   |
+
+You might have noticed that this fairly simple dataset will not capture *all* of the complexities that lead to a diagnosis of cardiovascular disease, but it might capture enough variability to build a model and make accurate predictions. Only experimentation will show if this dataset is capable of creating an accurate and generalizable model.
```
Comment (k8hertweck) [inline class1.md]: fairly simple dataset -> small collection of features

File: class1.md [hunk 20]
```diff
@@ -0,0 +1,261 @@
...
+
+**Inputs:** Features that we have collected and will use to make our prediction. Sometimes we will have datasets that have features that will not be included in downstream analysis. For example, in this dataset our inputs would be all of the columns except `patient_Id` and `cvd`. We would likely remove `pateint_Id` because a random identifier won't have any predictive power and `cvd` would not be included as an input since it is the target we are trying to predict.
+
+**Output:** The variable that we are trying to predict. In this case `cvd`.
+
+> #### Note!
+> The dataset above is from the package `cvdRiskData`. you can find more information [here](https://github.com/laderast/cvdRiskData). We will be using this dataset > as an example throughout this course.
+
+#### Challenge!
```
Comment (k8hertweck) [inline class1.md]: Same notes as above about challenge formatting

File: class1.md [hunk 21]
```diff
@@ -0,0 +1,261 @@
...
+3. Can we determine if an email is spam based on the text in the email?
+4. Given a database of images can we pull out all the distinct faces?
+
+[click here to go to solutions page](https://github.com/fredhutchio/concepts_machine_learning/blob/master/solutions/class1_solutions.md#determine-which-of-the-following-questions-are-supervised-and-unsueprvised-machine-learning-problems)
+
+## Wrapping up
+
+Today, we briefly explored some of the main ideas we will be covering throughout this course including exploratory data anlysis, dataset collection, and supervised and unsupervised learning concepts and methods. For further reading on these topics check out the optional materials below.
+
+Next week we will dive deeper into supervised learning. We will discuss more about data collection and how it relates to supervised learning, examine classifcation and regression methods, practice identifying and writing problem statements, and discuss common issues with supervised machine learning like over-fitting.
```
Comment (k8hertweck) [inline class1.md]: week -> class

File: class1.md (hunk 4 above)
Comment (k8hertweck) [reply class1.md:13]: also, ethics isn't specifically mentioned except in passing. We might think about reframing how this is emphasized throughout the course. It's important, but it's not fair to include it as a main objective when the content isn't there.

File: class2.md [hunk 22]
```diff
@@ -0,0 +1,287 @@
...
+
+Welcome to class 2 of Concepts in Machine Learning!
+
+In the last class, we touched on all of the main concepts that we will be covering in this course including supervised and unsupervised learning, exploratory data analysis, and ethics.
+
+By the end of this class you should be able to:
+
+* Define and understand the limitations of supervised learning
+* Differentiate between regression and classification
+* Assess whether or not supervised learning is an appropriate tool for a question
```
Comment (k8hertweck) [inline class2.md]: Isn't this generally a part of the first objective?

File: class2.md [hunk 23]
```diff
@@ -0,0 +1,287 @@
...
+* Understand basic applications of supervised learning (linear regression, logistic regression)
+* Create supervised learning problem statements
+
+### Review! What is machine learning?
```
Comment (k8hertweck) [inline class2.md]: Condense the review sections into a single subheading. For example, remove the actual data being referenced in the anatomy section.  I don't recommend exactly duplicating content between lessons, mostly because it's a huge pain if you have to update in more than one place. A paragraph at most should suffice.

File: class2.md [hunk 24]
```diff
@@ -0,0 +1,287 @@
...
+
+<p align="center">
+  <img width="350" alt="" src="images/histopathBreast.jpg">
+</p>
+
+* Input: Images of tumors
+* Output: Binary; begign or malignant
+* Dataset: Need a database of medical images, expert diagnosis of benign or malignant
+
+#### Identifying handwritten letters
```
Comment (k8hertweck) [inline class2.md:48]: Switch the order of these examples, so the non-biology (e.g, more general) example is first

File: class2.md [hunk 25]
```diff
@@ -0,0 +1,287 @@
...
+
+<p align="center">
+  <img width="400" alt="" src="images/meansqerror.png">
+</p>
+
+This is the most simple form of regression. The linear regression aims to fit a straight line onto the data that minimizes the mean squared error between the predicted value and the true output value. The mean squared error is the sum of the squared differences between the predictions and the true values, divided by the number of samples.
+
+You can also use polynomials to model curvature and interaction effects.
+
+Ordinary least squares regressions are very sensitive to outliars!
```
Comment (k8hertweck) [inline class2.md]: outliers

File: class2.md [hunk 26]
```diff
@@ -0,0 +1,287 @@
...
+
+<p align="center">
+  <img width="350" alt="" src="images/handwriting.png">
+</p>
+
+* Input: Scanned, handwritten letters
+* Output: The actual character
+* Dataset: Need thousands of handwritten letters and to annotate the correct letter for each one
+
+### How do you train and test a machine?
```
Comment (k8hertweck) [inline class2.md:70]: I think this class needs to include an example biomedical problem that we can use to frame each of the discussion points in subsequent sections. It doesn't have to be a real example, the one about disease prediction for which you already have a figure below would work. This would be a good topic for us to discuss at our Monday meeting

File: class2.md [hunk 27]
```diff
@@ -0,0 +1,287 @@
...
+* Linear regression: Aims to draw a straight best fit line through a field of data points. 
+* Polynomial regression: Aims to draw a curved best fit line through a field of data points.
+
+### Linear regression (AKA ordinary least squares)
+
+<p align="center">
+  <img width="400" alt="" src="images/meansqerror.png">
+</p>
+
+This is the most simple form of regression. The linear regression aims to fit a straight line onto the data that minimizes the mean squared error between the predicted value and the true output value. The mean squared error is the sum of the squared differences between the predictions and the true values, divided by the number of samples.
```
Comment (k8hertweck) [inline class2.md]: most simple -> simplest

File: class2.md [hunk 28]
```diff
@@ -0,0 +1,287 @@
...
+There are many different metrics that can be derived from a confusion matrix. Two of the most basic metrics are *accuracy* and the *error rate* of a classifier.
+* Accuracy: A measure of how often the machine guesses correctly
+  * `(TP+TN)/total = (100+50)/165 = 0.91`
+* Error rate: A measure of how often the machine guesses incorrectly
+  * `(FP+FN)/total = (10+5)/165 = 0.09`
+  * Equivalent to `1 - Accuracy`
+
+You could further interrogate the above problem by evaluating the sensitivity of the algorithm. Sensitivity measures the true positive rate and would show that the algorithm has 100% sensitivity for cat pictures and 0% sensitivity for dog pictures.
+
+### Regression
```
Comment (k8hertweck) [inline class2.md:204]: The structure of subheadings here doesn't make sense. It's not clear how the regression and linear regression sections fit into the overall structure of the class.

File: class2.md [hunk 29]
```diff
@@ -0,0 +1,287 @@
...
+* Mean Absolute Error (MAE)
+* R Squared (R²)
+* Adjusted R Squared (R²)
+* Mean Square Percentage Error (MSPE)
+* Mean Absolute Percentage Error (MAPE)
+* Root Mean Squared Logarithmic Error (RMSLE)
+
+Which metric you will use will be highly dependent on the context in which you're evaluating your algorithm.
+
+#### R Squared (R^2) is one of the simplest evaluations of a regression
```
Comment (k8hertweck) [inline class2.md]: Same issue here as above: I don't think these need to be separate subheadings, and should include more narrative to describe how they relate

File: class3.md [hunk 30]
```diff
@@ -0,0 +1,168 @@
+## Concepts in Machine Learning Class 3: Unsupervised Learning
+
+### Objectives
+
+Welcome to class 2 of Concepts in Machine Learning!
```
Comment (k8hertweck) [inline class3.md:5]: 2 -> 3

File: class2.md (hunk 24 above)
Comment (k8hertweck) [reply class2.md:48]: begign -> benign

File: class2.md [hunk 31]
```diff
@@ -0,0 +1,287 @@
...
+The main difference between classification and regression is that classification predicts a categorical variable and regression predicts a continuous numerical value.
+
+<p align="center">
+  <img width="350" alt="" src="images/supervised.png">
+</p>
+
+### Classification
+
+Classification aims to predict a class label, which is a choice from a predetermined list of possibilities.
+* Binary classification: Classifying into exactly two classes.
```
Comment (k8hertweck) [inline class2.md]: collapse into descriptive sentence

File: class2.md [hunk 32]
```diff
@@ -0,0 +1,287 @@
...
+* Multiclass classification: Classifying into more than two classes.
+
+*Decision trees* and *support vector machines* are two kinds of classification methods.
```
Comment (k8hertweck) [inline class2.md]: these are different from each other in <these ways> (or define each), focusing now on decision trees, because....

File: class2.md [hunk 33]
```diff
@@ -0,0 +1,287 @@
...
+* false negatives (FN): We predicted no, but they actually do have the disease. (Also known as a "Type II error.")
+
+There are many different metrics that can be derived from a confusion matrix. Two of the most basic metrics are *accuracy* and the *error rate* of a classifier.
+* Accuracy: A measure of how often the machine guesses correctly
+  * `(TP+TN)/total = (100+50)/165 = 0.91`
+* Error rate: A measure of how often the machine guesses incorrectly
+  * `(FP+FN)/total = (10+5)/165 = 0.09`
+  * Equivalent to `1 - Accuracy`
+
+You could further interrogate the above problem by evaluating the sensitivity of the algorithm. Sensitivity measures the true positive rate and would show that the algorithm has 100% sensitivity for cat pictures and 0% sensitivity for dog pictures.
```
Comment (k8hertweck) [inline class2.md]: update dog/cat

File: class1.md (hunk 3 above)
Comment (lakikowolfe) [reply class1.md]: I'll just remove it. I'd rather keep it shorter/more simple

**Review by lakikowolfe — COMMENTED** [review]
Incorporated Kates comments into class1.md and class2.md

**Comment by k8hertweck** [comment]
Merging now so I can clean up file structure, etc
